akka-actor/src/main/scala/akka/util/Helpers.scala

```diff
@@ -130,12 +130,20 @@ object Helpers {
    * INTERNAL API
    */
   private[akka] final implicit class ConfigOps(val config: Config) extends AnyVal {
+
+    import JavaDurationConverters._
+
     def getMillisDuration(path: String): FiniteDuration = getDuration(path, TimeUnit.MILLISECONDS)
 
     def getNanosDuration(path: String): FiniteDuration = getDuration(path, TimeUnit.NANOSECONDS)
 
     private def getDuration(path: String, unit: TimeUnit): FiniteDuration =
       Duration(config.getDuration(path, unit), unit)
+
+    def getPotentiallyInfiniteDuration(path: String): Duration = toRootLowerCase(config.getString(path)) match {
+        case "infinite" | "" | "off" => Duration.Inf
```

Comment:
why not `Undefined` for `"" | "off"`?
we use Undefined in some places, e.g. https://github.com/akka/akka/blob/master/akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala#L89

Reply:
I misunderstood.